lib/pages/v1_app.dart

```diff
@@ -394,6 +394,35 @@ class _V1AppState extends State<V1App> {
                     ),
                   ),
                 ),
+                TextButton(
+                  onPressed: () {
+                    if (pageNumber != Pages.Vote) {
+                      setPageNumber(Pages.Vote);
+                    }
+                  },
+
+                  child: Text(
+                    'Vote',
+                    style: textSwapState(
+                      pageNumber == Pages.Vote,
+                      textStyle(
+                        Colors.white,
+                        tabTxSz,
+                        true,
+                        false,
+                      ),
+                      textStyle(
+                        Colors.amber[400]!,
+                        tabTxSz,
+                        true,
+                        true,
+                      ),
+                    ),
+                  ),
+                ),
+
+
+
```

Comment:
This is not a new page feature. It should just be a `TextButton` with a child Text Widget that says 'Vote' and the `onPressed` callback should take the user to the link that was provided in the ticket description.